src/dbwrapper.h

```diff
@@ -44,6 +45,11 @@ const std::vector<unsigned char>& GetObfuscateKey(const CDBWrapper &w);
 
 };
 
+class CBitcoinLevelDBLogger : public leveldb::Logger {
```

Comment:
Why is this class in the .h file? It seems to be only used internally in dbwrapper.cpp.